nomad/core_sched.go

```diff
@@ -484,17 +484,17 @@ OUTER:
 	c.logger.Debug("node GC found eligible nodes", "nodes", len(gcNode))
 
 	// Call to the leader to issue the reap
-	for _, nodeID := range gcNode {
+	for _, ids := range partitionAll(maxIdsPerReap, gcNode) {
```

Comment:
Use ServersMeetMinimumVerison("0.9.3") to determine whether or not to batch and use a new API+Raft message.